lib/ProductOpener/Text.pm

```diff
@@ -146,10 +146,10 @@ sub _get_locale_percent_regex {
 	# [+-]?(?:\d{3}\.)*\d+(?:,\d+)*\h*% where . is the group sign from the locale, and , is the decimal point - or other way around for tr etc.
 	my $regex;
 	if (index($perf->pattern, $perf->percent_sign) == 0) {
-		$regex = qr/(%\h*[$p$m]?(?:\d{3}$g)*\d+(?:$d\d+)*)/;
+		$regex = qr/(%\h*[$p$m]?(?:\d{1,3}$g)*\d+(?:$d\d+)*)/;
```

Comment:
Why this change? For which strings is it needed?

Reply:
`is( normalize_percentages('test 0,12,345.67% hey', 'ur'), 'test 12,345.67% hey' );`

`is( normalize_percentages('test 1.23.045,67% hey', 'de'), "test 123.045,67\N{U+00A0}% hey" );`

For these test cases, where the number of digits before the `group_sign` can be 2 or 1 .

Reply:
I thought digits were only grouped by groups of 3, but from reading https://en.wikipedia.org/wiki/Decimal_separator#Digit_grouping it seems that in India it is first a group of 3 then groups of 2.... 
In practice we won't get many percent values that are more to 100%, but for comparisons it could happen.
We don't want 2,50 to be converted to 250 (even in English).
Maybe we can change the regexp so that the last group of digits is \d{3}.